src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -550,7 +550,12 @@ class DetailsCard extends React.Component {
 
     // ---- dispatch the save
     this.setState({ correlationId })
-    this.props.saveChanges(vmUpdates, this.restartAfterSave, !this.hotPlugNow, correlationId)
+    this.props.saveChanges(
+      vmUpdates,
+      this.restartAfterSave,
+      !this.hotPlugNow,
+      isVmRunning,
```

Comment:
Is this parameter really needed, because you can change CD only if VM is running, so for CD this parameter always will be `true`?

Reply:
While strictly true, change CD is only currently enabled for a running VM, the code made more sense to me with the param specified in there.  Putting the param made sense at the time.

Reply:
I'll take it out.  It's less confusing that way after I had some more coffee.